### ISSUE TITLE
No highlighting of current page in the sidebar navigation

### ISSUE THREAD

**Issue by kitation** [UKHomeOffice/design-system#235]
There should be a visual indicator as to which page a user is on in the sidebar navigation. Also all the links have aria-current set to true which makes the screen reader say every page is the current page.
